contracts/handler/CompatibilityFallbackHandler.sol

```diff
@@ -20,6 +21,12 @@ contract CompatibilityFallbackHandler is TokenCallbackHandler, ISignatureValidat
      */
     bytes32 private constant SAFE_MSG_TYPEHASH = 0x60b3cbf8b4a223d68d641b3b6ddf9a298e7f33710cf3d3a9d1146b5a6150fbca;
```

Comment:
LGTM, but can we further improve these and extract the relevant constants to a library? I'd imagine it'd be useful to 3rd party developers and also reduce the duplication between core contracts and the handler

Reply:
Tracked in https://github.com/safe-global/safe-smart-account/issues/974

I think a refactor would be outside the scope of the audit.